frontend/src/components/ManageAccess.vue

```diff
@@ -32,56 +34,56 @@ const userRoleAssignments = ref<UserRoleAssignment[]>([])
 
 // TODO: Eliminate test data
 userRoleAssignments.value = [
-    {
-        "user_role_xref_id": 1,
-        "user": {
-          "user_type": {
-            code: "I",
-            description: "IDIR",
-          },
-          "user_name": "foo-test"
-        },
-        "role": {
-            "role_name": "Reviewer",
-            "role_type_code": "C"
-        }
+  {
+    "user_role_xref_id": 1,
+    "user": {
+      "user_type": {
+        code: "I",
+        description: "IDIR",
+      },
+      "user_name": "foo-test"
```

Comment:
Looks like you are reverting a change I need to keep.

Reply:
I am sorry about that, I must have reverted by mistake when fetching the code from server.